UCLA Radio/UCLA Radio/fonts/DesignableLabel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DesignableLabel.swift
+//  UCLA Radio
+//
+//  Created by Chris on 2/15/17.
+//  Copyright © 2017 UCLA Student Media. All rights reserved.
+//
+
+import UIKit
+
+/*
+ Fixes storyboard bug where custom fonts with attributed strings
+ don't work correctly with UILabel
+
+ http://stackoverflow.com/questions/25925914/attributed-string-with-custom-fonts-in-storyboard-does-not-load-correctly
+ 
```

Comment:
Nice! I noticed that that the about page didn't bold, but wasn't sure how to fix that. Good find!